fe/fe-core/src/main/java/com/starrocks/qe/SessionVariable.java

```diff
@@ -845,6 +848,15 @@ public static MaterializedViewRewriteMode parse(String str) {
     @VarAttr(name = SPILL_ENCODE_LEVEL)
     private int spillEncodeLevel = 7;
 
+    @VarAttr(name = ENABLE_AGG_SPILL_PREAGGREGATION, flag = VariableMgr.INVISIBLE)
+    public boolean enableAggSpillPreaggregation = true;
+
+    @VarAttr(name = AGG_SPILL_HT_LOW_REUCTION_THRESHOLD, flag = VariableMgr.INVISIBLE)
+    public double aggSpillHtLowReductionThreshold = 0.5;
+
+    @VarAttr(name = AGG_SPILL_HT_LOW_REDUCTION_CHUNK_LIMIT, flag = VariableMgr.INVISIBLE)
+    public int aggSpillhtLowReductionChunkLimit = 5;
```

Comment:
need adjust these ? if yes, use force is easier?

Reply:
> need adjust these ? if yes, use force is easier?

maybe we don't need expose these configurations to user, I will remove them